programmers/신규아이디.js

```diff
@@ -0,0 +1,26 @@
+function solution(newID) {
+    
+    let renewID = newID.toLowerCase()                               //1st step done
+                       .replace(/[^\w\d\-\_\.]/g, '')               //2nd step done
+                       .replace(/\.{2,}/g, '.')                     //3rd step done
+                       .replace(/^\.|\.$/g, '');                    //4th step done
+    
+    if (!renewID) {
+        for(let i = 0; i < newID.length; i++)                       //5th step done
+            renewID += "a";
+    }
```

Comment:
```suggestion
    if (!renewID.length) renewID = "aaa"; 
```
이렇게써줘도 될꺼같아요
그리고 renewID말고 renewId하는게 좋은 변수명 같습니다